services/cd-service/pkg/repository/repository.go

```diff
@@ -2612,6 +2616,13 @@ func (s *State) GetApplicationReleaseFromManifest(application string, version ui
 		return nil, err
 	}
 	release.UndeployVersion = isUndeploy
+	release.IsMinor = true
+	if _, err := readFile(s.Filesystem, s.Filesystem.Join(base, "minor")); err != nil {
```

Comment:
Is it really necessary to read this here? I would be fine if this feature only worked with the DB enabled.